library/std/src/sys/solid/env.rs

```diff
@@ -0,0 +1,9 @@
+pub mod os {
+    pub const FAMILY: &str = "itron";
+    pub const OS: &str = "solid";
+    pub const DLL_PREFIX: &str = "";
+    pub const DLL_SUFFIX: &str = "";
+    pub const DLL_EXTENSION: &str = "";
```

Comment:
Not `.so` or something like that?

Reply:
Addressed in 72fcf04737f40d6e2c12dda7a8de9248ca031ef0.